packages/frontend/src/components/wallet-migration/WalletMigration.jsx

```diff
@@ -91,22 +92,30 @@ const WalletMigration = ({ open, onClose }) => {
         handleSetActiveView(WALLET_MIGRATION_VIEWS.VERIFYING);
     };
 
-    const navigateToCleanKeys = () => {
-        handleSetActiveView(WALLET_MIGRATION_VIEWS.CLEAN_KEYS);
-    };
-
     const navigateToLogOut = () => {
         setMigrationStep(WALLET_MIGRATION_VIEWS.LOG_OUT);
         handleSetActiveView(WALLET_MIGRATION_VIEWS.LOG_OUT);
     };
 
+    const navigateToMigrateAccounts = () => {
+        handleSetActiveView(WALLET_MIGRATION_VIEWS.MIGRATE_ACCOUNTS);
+    };
+
     const onLogout = () => {
-        // TODO: Add logic to remove FAK(s) here so everything gets cleared together
-        return Promise.all(availableAccounts.map((accountId) => wallet.removeWalletAccount(accountId)))
+        setIsLoggingOut(true);
+        return Promise.all(availableAccounts.map(async (accountId) => {
```

Comment:
If an account fails in here, does the UI break (i.e. does `isLoggingOut` remain `true`)? I'm wondering if a `.reflectAll` might be better here to aggregate failed removals - at least displaying an error for the accounts that failed.

Realistically I think this only fails during RPC connection issues so it's not super pressing for this PR IMO.